site/docs/v1/tech/lambdas/epic-games-reconcile.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Epic Games Reconcile
+description: An overview of the Epic Games Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Epic Games Reconcile lambda
+
+When an Epic Games identity provider is used to complete a federated login request FusionAuth. Please note, there are multiple linking strategies when attempting to reconcile the user. Epic Games defaults to `CreatePendingLink`. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement.
+
+If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function reconcile(user, registration, idToken) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
+
+* `user` - the FusionAuth User object
+* `registration` - the FusionAuth UserRegistration object
+* `accessToken` - the JSON payload returned by the Epic Token API
+
+The two FusionAuth objects are well documented here in the link:/docs/v1/tech/apis/users/[User API] and link:/docs/v1/tech/apis/registrations/[Registration API] documentation. The `idToken` may contain various user claims depending upon the user's Epic Games configuration.
+
+== Assigning the lambda
+
+Once a lambda is created, you may assign it to the Epic Games identity provider in the IdP configuration.
+
+Navigate to [breadcrumb]#Settings -> Identity Providers# and select your existing Epic Games configuration or click [breadcrumb]#Add provider# and select Epic Games if it has not yet been configured.
+
+== Example lambda
+
+There is not a default Epic Games reconcile lambda provided by FusionAuth. The following is an example lambda you can use as a starting point.
+
+[source,javascript]
+----
+// Using the response from the Epic API, reconcile the User and User Registration.
+function reconcile(user, registration, accessToken) {
+  //  When writing a lambda we've added a few helpers to make life easier.
+  //  console.info('Hello World');         # This will create an EventLog of type Information
+  //  console.error('Not good.');          # This will create an EventLog of type Error
+  //  console.debug('Step 42 completed.'); # This will create an EventLog of type Debug
+  //
+  //  To dump an entire object to the EventLog you can use JSON.stringify, for example:
+  //  console.info(JSON.stringify(user));
+
+  // Happy coding! Reconcile the User here.
+
+  console.info('Hello World!');
```

Comment:
Same comment as with the Client Credentials, if we provide an example at all, it should probably be a real example. Otherwise we can omit this since we already provide the method signature earlier.